src/components/Calculator.js

```diff
@@ -1,40 +1,54 @@
-import React from 'react';
-import CalculatorKey from './CalculatorKey';
+import React, { useState } from 'react';
+// import button from './button';
 import '../Calculator.css';
+import calculate from '../logic/calculate';
 
-const Calculator = () => (
-  <section className="calculator">
-    <div className="calculator-output">
-      <div className="calculator-screen">0</div>
-    </div>
-    <div className="Calculator-buttons">
-      <div className="calculator-functions">
-        <CalculatorKey BtnValue="AC" />
-        <CalculatorKey BtnValue="+/-" />
-        <CalculatorKey BtnValue="%" />
-      </div>
-      <div className="calculator-operators">
-        <CalculatorKey BtnValue="/" />
-        <CalculatorKey BtnValue="x" />
-        <CalculatorKey BtnValue="-" />
-        <CalculatorKey BtnValue="+" />
-        <CalculatorKey BtnValue="=" />
+const Calculator = () => {
+  const [state, setState] = useState({
+    total: 0,
+    next: null,
+    operation: null,
+  });
+
+  const handleClick = (event) => {
+    setState((state) => calculate(state, event.target.innerText));
+  };
+
+  const { total, next } = state;
+  const display = next || total || '0';
+  return (
+    <section className="calculator">
+      <div className="calculator-output">
+        <div className="calculator-screen">{display}</div>
```

Comment:
- [ ] When trying to do a divide operation like 2 / 2  it gives the following error please make sure to fix it in order to complete the requirements 
<img width="1407" alt="Screenshot 2023-04-27 at 21 22 21" src="https://user-images.githubusercontent.com/61790587/234982474-e967f96b-199f-47cb-a320-0f82dc493d79.png">